lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -678,7 +679,7 @@ def _delete_environment(self, environment: Environment, log: Logger) -> None:
             else:
                 log.debug("not wait deleting")
 
-    def _save_console_log(
+    def _check_panic(
```

Comment:
The method name should also specify it saves the serial log.

Reply:
updated